src/main/java/org/eclipse/basyx/submodel/metamodel/map/submodelelement/SubmodelElementCollection.java

```diff
@@ -108,6 +108,8 @@ public static SubmodelElementCollection createAsFacade(Map<String, Object> obj)
 		}
 		
 		if (!isValid(obj)) {
+			System.out.println("Exception : obj = " + obj);
```

Comment:
Please remove these two prints to command line

Reply:
Removed this two print lines.